ibm/resource_ibm_function_action.go

```diff
@@ -145,17 +160,67 @@ func resourceIBMFunctionAction() *schema.Resource {
 				Computed:    true,
 				Description: "All paramters set on action by user and those set by the IBM Cloud Function backend/API.",
 			},
+			"action_id": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
You defined the action_id but d.Set the value is not set  simialrly for other resources also can you check